README.md

```diff
@@ -78,6 +78,7 @@ Inspired by [Custom UI: Mini media player](https://community.home-assistant.io/t
 | sound_mode | string | optional | v1.1.2 | Change sound mode select appearance, `icon` for just an icon, `full` for the full sound mode name.
 | info | string | optional | v1.0.0 | Change how the media information is displayed, `short` to limit media information to one row, `scroll` to scroll overflowing media info.
 | volume_stateless | boolean | false | v0.6 | Swap out the volume slider for volume up & down buttons.
+| volume_step | string | 10 | v1.9.0 | Change the volume step size of volume buttons.
```

Comment:
Looking good!
Please just change this from `10` to `optional` and we are good to merge this!

Reply:
Could probably also change `string` to `number`.

Reply:
https://github.com/kalkih/mini-media-player/pull/308/commits/c23ef266b5e4507ba24010356e4518f5bd6782fa done :)